apps/dashboard/src/app/team/[team_slug]/(team)/~/projects/TeamProjectsPage.tsx

```diff
@@ -132,6 +142,27 @@ export function TeamProjectsPage(props: {
         </div>
       )}
 
+      {showExpandCollapseButton && (
```

Comment:
if you're going to do this do this on the server side so that we actually send less data down initially, right now ur sending all the data down and then just filtering on the client

Reply:
It feels much better/responsive if filtering is done on client side - Doing this on server side only makes it slower